nelsk-kore/lib/kore/comm/ingress_message.rb

```diff
@@ -0,0 +1,16 @@
+module Kore
+  module Comm
+    class IngressMessage
+      attr_accessor :originator, :plugin, :cmd, :content, :raw
```

Comment:
I don't see raw being used. Is it supposed to exist?

Reply:
Very good point. `parse` is defined by the concrete adapter, because the messages are going to very based on where they're coming from. I'd like to follow up with an adjustment that demo's that truth.

Reply:
Ack. Missed pruning this during refactoring. FYI, the  `{Ingress, Egress}Message`s I see expanding beyond just data structs. I wouldn't take these to be authoritative. I need to adjust the README to reflect that.